src/relay.js

```diff
@@ -288,12 +292,12 @@ export function sequelizeConnection({
         // In case of `OVER()` is not available, we need to get the full count from a second query.
         const options = await Promise.resolve(before({
           where: argsToWhere(args)
-        }));
+        }, args, context, info));
 
         if (target.count) {
-          fullCount = await target.count(source, options);
+          fullCount = await target.count(options);
         } else {
-          fullCount = await target.manyFromSource.count(source, options);
+          fullCount = await target.manyFromSource.count(options);
```

Comment:
Are you talking about this one line 300 ? Or the ones above also ?
Because without theses fixes, the connection doesn't work. But maybe the change line 300 was unnecessary.